src/sql/azdata.proposed.d.ts

```diff
@@ -810,10 +810,25 @@ declare module 'azdata' {
 	}
 
 	export interface TableComponent {
-
 		/**
 		 * Append data to an exsiting table data.
 		 */
 		appendData(data: any[][]);
 	}
+
+	export interface IconColumnCellValue {
+		icon: string | vscode.Uri | { light: string | vscode.Uri; dark: string | vscode.Uri };
+		ariaLabel: string;
```

Comment:
the name is correct here, but other places, you are calling it areaLabel 🤣

Reply:
fixed.